internal/controller/gateway/suite_test.go

```diff
@@ -22,7 +22,7 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 */
 
-package gateway
+package controller
```

Comment:
@jonstacks I'm curious your thoughts here.

Currently we put everything under `internal/controller` as `package controller`, but I could see us using `package ingress` for `internal/controller/ingress` etc... for other sub-pkgs.
Wdyt?

Reply:
I think that was probably just due to the original reorganization of the files and moving them from the internal folder to an `ingress` folder. My 2c is to stick with `gateway` for this package, the ingress controller specifically becomes `package ingress`, and then we import them as:

```go
import (
        ingresscontroller "github.com/ngrok/ngrok-operator/internal/controller/ingress"
        gatewaycontroller "github.com/ngrok/ngrok-operator/internal/controller/gateway"
)
```